slackclient/_slackrequest.py

```diff
@@ -76,4 +77,9 @@ def do(self, token, request="?", post_data=None, domain="slack.com", timeout=Non
         post_data['token'] = token
         headers = {'user-agent': self.get_user_agent()}
 
-        return requests.post(url, headers=headers, data=post_data, files=files, timeout=timeout)
+        return requests.post(url,
+                             headers=headers,
+                             data=post_data,
+                             files=files,
+                             timeout=timeout,
+                             proxies=self.proxies)
```

Comment:
the `proxies` dict that's used to initialize `SlackClient` gets passed to both `requests.post` and `websocket.create_connection`, so this affects both web API & websocket @Roach